README.md

```diff
@@ -335,6 +336,28 @@ the name of the virtual environment created by poetry.
 - Enter inside the docker file: `docker compose exec full_text_searcher /bin/bash`
 - Running the scripts: `docker compose exec full_text_searcher python ht_full_text_search/export_all_results.py '"justice blame"'`
 
+### Guides to install python and poetry on macOS
+
+Recommendation: Use brew to install python and pyenv to manage the python versions.
+
+* Install python
+    * You can read this blog to install python in the right way in
+      python: https://opensource.com/article/19/5/python-3-default-mac
+* Install poetry:
+    * **Good blog to understand and use poetry
+      **: https://blog.networktocode.com/post/upgrade-your-python-project-with-poetry/
+    * **Poetry docs**: https://python-poetry.org/docs/dependency-specification/
+    * **How to manage Python projects with Poetry
+      **: https://www.infoworld.com/article/3527850/how-to-manage-python-projects-with-poetry.html
+
+* Useful poetry commands (Find more information about commands [here](https://python-poetry.org/docs/cli))
+    * Inside the application folder: See the virtual environment used by the application `` poetry env use python ``
+    * Activate the virtual environment: ``source ~/ht-indexer-GQmvgxw4-py3.11/bin/activate``, in Mac poetry creates
```

Comment:
Why the random characters (`GQmvgxw4`) in the directory name?

I would probably suggest only listing the poetry commands needed to run this project rather than listing how to use poetry generally (although I think the link to poetry documentation in general is useful)

Reply:
The directory **ht-indexer-GQmvgxw4** is automatically created by poetry, then it concatenates the name of the root folder with an alphanumeric sequence to name the directory.

I'll update the documentation adding only the poetry commands to set up the virtual environment of this project. I can add a section with additional information about poetry.